gcb/fast-forward/cloudbuild.yaml

```
@@ -0,0 +1,78 @@
+timeout: 14400s
+
+#### SECURITY NOTICE ####
+# Google Cloud Build (GCB) supports the usage of secrets for build requests.
+# Secrets appear within GCB configs as base64-encoded strings.
+# These secrets are GCP Cloud KMS-encrypted and cannot be decrypted by any human or system
+# outside of GCP Cloud KMS for the GCP project this encrypted resource was created for.
+# Seeing the base64-encoded encrypted blob here is not a security event for the project.
+#
+# More details on using encrypted resources on Google Cloud Build can be found here:
+# https://cloud.google.com/cloud-build/docs/securing-builds/use-encrypted-secrets-credentials
+#
+# (Please do not remove this security notice.)
+secrets:
+- kmsKeyName: projects/k8s-releng-prod/locations/global/keyRings/release/cryptoKeys/encrypt-0
+  secretEnv:
+    GITHUB_TOKEN: CiQAIkWjMFgubym04n054BpfS8ED1VAiDVzF2WtEAE9kpXL1HEASUQBLz1D++TFNeowLVAI8MAQ86xJvshtqevcU5lcHEo3K+39HHIZr3uojPto5vm235XkfWWeMM3/VBOwZoelmAyvY2RPHpo7u/Y9kKfCrJu5coA==
```

This is a [previous version of the stage](https://github.com/kubernetes/release/commit/86a6ee33d67277815627baf5eb60d2edbbe693a2) token that does not have write access to k/k, use the [release token](https://github.com/kubernetes/release/blob/d66b62751a23dca58fbdf9a2cab400e471fa734a/gcb/release/cloudbuild.yaml#L17) instead to be able to push to GitHub:

```suggestion
    GITHUB_TOKEN: CiQAIkWjMODH+TR8+6KmsAValHBKkX2PjVWSzyNdScGCIl0rBlYSUQBLz1D++mrhyd+wXohoQEGDuyQHvFzQyT7NEvI2BR/WBmnu1S9+E5GgER8OOnaQg+dGEMrYzEQIiTOB8WPxU7ecsIyo0AuegcBH24dj9L2tDw==
```

---

Good catch! Changed as suggested.